contrib/endpoints/src/api_manager/config_manager.cc

```diff
@@ -47,6 +47,11 @@ ConfigManager::ConfigManager(
     }
   }
 
+  if (global_context_->server_config()->has_init_service_configs()) {
```

Comment:
Do you want to add set_current_rollout_id() to ConfigManager and call it in ApiManager?

I didn't include using ConfigManager in ApiManager this PR to reduce the size of the code review.